implementation/routing/src/routing_manager_base.cpp

```diff
@@ -1184,8 +1184,8 @@ void routing_manager_base::remove_local(client_t _client,
         std::lock_guard<std::mutex> its_lock(local_services_mutex_);
         // Finally remove all services that are implemented by the client.
         std::set<std::pair<service_t, instance_t>> its_services;
-        for (auto& s : local_services_) {
-            for (auto& i : s.second) {
+        for (auto const& s : local_services_) {
+            for (auto const& i : s.second) {
```

Comment:
Prefer `const auto` over `auto const`.

```suggestion
        for (const auto& s : local_services_) {
            for (const auto& i : s.second) {
```